src/main/webapp/script.js

```diff
@@ -10,4 +10,32 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function getYouTubeComments() { 
+  const dataContainer = document.getElementById('data-container');
+  const urlInput = document.getElementById('url-entry');
+  var URL = cleanseUrl(urlInput.value);
+  fetch("/YouTubeComments?url="+URL)
+    .then(response => response.json()).then((comments) =>{
+      // Generate comments
+      dataContainer.innerText = "Howdy world";
+
+      for(i = 0; i < comments.items.length; i++) {
+        dataContainer.appendChild(createYouTubeCommentElement(comments.items[i]));
+      }
+    });
+}
+
+// Extracts video id from full url
+function cleanseUrl(url) {
+  // Split web address from parameters, extract first parameter
+  var videoId = url.split("?");
+  videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
+
+  // Remove parameter name to isolate video Id.
+  if(videoId.includes("v=")) { 
```

Comment:
Is there any chance the video url could contain this?   If not, all good.

Reply:
By this I mean "v=", as part of the ID

Reply:
I do not believe so